src/test/java/com/vackosar/gitflowincrementalbuild/mocks/EmptyLocalRepoMock.java

```diff
@@ -0,0 +1,46 @@
+package com.vackosar.gitflowincrementalbuild.mocks;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+
+/**
+ * Local git repo, without any commits.
+ * 
+ * @author mikr
```

Comment:
This should either be your GH account or just skip it entirely (see most of the other classes).

Reply:
removed.